src/coreComponents/physicsSolvers/surfaceGeneration/SurfaceGenerator.cpp

```diff
@@ -726,8 +726,9 @@ int SurfaceGenerator::separationDriver( DomainPartition & domain,
   real64 ruptureRate = calculateRuptureRate( elementManager.getRegion< SurfaceElementRegion >( this->m_fractureRegionName ) );
 
   GEOS_LOG_LEVEL_RANK_0( 3, "rupture rate is " << ruptureRate );
-  if( ruptureRate > 0 )
-    m_nextDt = ruptureRate < 1e99 ? m_cflFactor / ruptureRate : 1e99;
+  if( ruptureRate > 0 ) //FIXME ugly adaptation while waiting to know if we have to have a new spec flow value or can factorize.
```

Comment:
@jafranc what is going on here? :)

Reply:
lol, I restored the `develop` version I was highjacking but forgot to erase my reminder :p